### PR TITLE
initialized rails

### PR DESCRIPTION
**Pull request by magnus-thor**
## PT link: https://www.pivotaltracker.com/story/show/151371845 ##
------------------

- Initialized new rails app
### What have I learned ###
- I learned how to create a rails app within a folder
- I learned to remember to follow instructions

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: I would clean up this file

File: README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```
Comment (tochman) [inline README.md]: I would make sure that the README is restored to include info about your project

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: laundry_time_production
+  username: laundry_time
+  password: <%= ENV['LAUNDRY_TIME_DATABASE_PASSWORD'] %>
```
Comment (tochman) [inline config/database.yml:17]: I would clean up this file...

**Review by Sajaas — APPROVED** [review]
Lots of comments in the files, but most of them seem useful to us noobs.

**Pull request merged**